install.sh

```diff
@@ -470,6 +476,15 @@ for i in seastar/scripts/perftune.py seastar/scripts/seastar-addr2line; do
 done
 relocate_python3 "$rprefix"/scyllatop tools/scyllatop/scyllatop.py
 
+if $debuginfo; then
+    install -d -m755 "$rprefix"/libexec/.debug
+    cp -r ./debug/libexec/* "$rprefix"/libexec/.debug
```

Comment:
Did you check that debugging works with this? In https://sourceware.org/gdb/onlinedocs/gdb/Separate-Debug-Files.html they say the files need to be named `.debug/scylla.debug`.

Reply:
The script copying following .debug files to /opt/scylladb/libexec/.debug,
```
$ ls -l build/dist/release/debuginfo/scylla/usr/lib/debug/libexec/
合計 1933320
-r--r--r--. 1 syuu docker   66950416  7月 12 04:12 iotune-4.6.dev-0.20210712.e2365b562.x86_64.debug
-r--r--r--. 1 syuu docker 1912763872  7月 12 04:13 scylla-4.6.dev-0.20210712.e2365b562.x86_64.debug
```
So it will be like /opt/scylladb/libexec/.debug/ scylla-4.6.dev-0.20210712.e2365b562.x86_64.debug.

The additional prefix (-4.6.dev-0.20210712.e2365b562.x86_64.)  is comming from `/usr/lib/rpm/find-debuginfo.sh  --unique-debug-suffix -$SCYLLA_VERSION-$SCYLLA_RELEASE.$(arch)`, I think the script modifies stripped binary to point out suffix added debug symbol file.

Reply:
So we also can use the filename `.debug/scylla.debug` by dropping --unique-debug-suffix from find-debuginfo.sh